src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    private String color;
```

Comment:
```suggestion
    private Color color;
```
Unlike using String for color property it insures against having a figure with the color "My cute cat"